### PR TITLE
Development branch merge

### PR DESCRIPTION
**Pull request by appakz**
@jamescrowley321 this PR is for getting the changes you've been making in Development merged into master so they can be deployed. I'll review them now and then do the merge if it looks good.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -222,3 +222,4 @@ Event-Site/packages/RavenDB.Server.2.0.2330/tools/Database/
 
 #ImageStorage
 /EventSite/Content/localImageStorage/
+/.vs/config
```
Comment (appakz) [inline .gitignore]: we shiould just ignore at the root .vs level, which I added earlier, but this is OK for now.

File: EventSite/Domain/Model/Sponsor.cs [hunk 2]
```diff
@@ -16,6 +16,11 @@ public class Sponsor {
         public SponsorStatus Status { get; set; }
         public DateTimeOffset? DonatedOn { get; set; }
 
+        public Sponsor()
+        {
+            this.Link = "http://";
```
Comment (appakz) [inline EventSite/Domain/Model/Sponsor.cs]: @jamescrowley321 what's up with this?

Comment (jamescrowley321) [inline EventSite/Domain/Model/Sponsor.cs]: Keith was complaining about having to put the "http://" in the field for sponsors. My plan was to add server side validation, just ran out of time today. You can take it out it and I'll put something legit in tomorrow.


Sent from Outlook






On Tue, Feb 9, 2016 at 6:16 PM -0800, "Jesse Taber" <notifications@github.com> wrote:












In EventSite/Domain/Model/Sponsor.cs:
> @@ -16,6 +16,11 @@ public class Sponsor {
>          public SponsorStatus Status { get; set; }
>          public DateTimeOffset? DonatedOn { get; set; }
>  
> +        public Sponsor()
> +        {
> +            this.Link = "http://";



@jamescrowley321 what's up with this?

File: EventSite/Views/Home/Index.cshtml [hunk 3]
```diff
@@ -5,7 +5,16 @@
     ViewInfo.Title = "Home";
 }
 
-@Html.Partial("_Carousel")
+@*@Html.Partial("_Carousel")*@
+
+    <!-- Header -->
```
Comment (appakz) [inline EventSite/Views/Home/Index.cshtml]: I don't think we need the "header" comment here; it's pretty obvious.

File: EventSite/Views/Location/Index.cshtml [hunk 4]
```diff
@@ -97,18 +98,27 @@
...
         }
     </div>
-
-    @if (Model.Venue.HasAddressSet())
-    {
-        <div class="col-md-9" role="main">
-            <div id="map" style="width: 100%; height: 480px;" />
+    <div class="col-md-9">
+        <div class="text-center">
+            @*<h3><a href="/Content/downloads/TallahasseeCodeCamp2014Schedulev1.5.pdf" target="_blank">Download Session Schedule</a></h3>*@
```
Comment (appakz) [inline EventSite/Views/Location/Index.cshtml]: The link to last year's version of the PDF schedule shouldn't be in here.

File: EventSite/Views/Speakers/Index.cshtml [hunk 5]
```diff
@@ -14,9 +14,9 @@
 
     <div class="row featurette">
         <a data-toggle="modal" href="#keynote-speaker" class="">
-            <div class="col-md-7" style="text-align: center;"><h2 class="featurette-heading">Keynote Speaker: <br />Ryan Kopinsky</h2></div>
+            <div class="col-md-7" style="text-align: center;"><h2 class="featurette-heading">Keynote Speaker: <br />Coming Soon!</h2></div>
             <div class="col-md-5">
-                <img class="decorative-image challenge img-responsive" style="width: 200px; height:200px" src="~/Content/images/home/carousel/keynote_speaker.JPG" alt="Ryan Kopinsky" />
+                @*<img class="decorative-image challenge img-responsive" style="width: 200px; height:200px" src="~/Content/images/home/carousel/keynote_speaker.JPG" alt="Ryan Kopinsky" />*@
```
Comment (appakz) [inline EventSite/Views/Speakers/Index.cshtml]: Don't just comment this out; delete it.

**Comment by appakz** [comment]
@jamescrowley321 made a few notes, but nothing worth holding up this PR for. Please address the other stuff I mentioned when you can.

**Comment by jamescrowley321** [comment]
Issues are resolved in development branch and will make there way into the
master in the PR. I'll knock out some of the other issues during the
holiday next week.

**Pull request merged**